### ISSUE TITLE
Active Monitoring not working on Debian Bullseye distributed package

### ISSUE THREAD

**Issue by alexolivan** [ntop/ntopng#8386]
**Environment**:
* OS name: Debian
* OS version: Bullseye
* Architecture: amd64
* ntopng version/revision: ntopng Community v.6.1.240509 (Debian GNU/Linux 11 (bullseye))

**What happened**:
AM does nothing.
Log states /usr/share/ntopng/scripts/lua/modules/am_utils.lua failure: round() function is not defined

**Workaround**:
Added at the top of the script the folling:

```
require "lua_utils"
local format_utils = require "format_utils"
```

**How did you reproduce it?**
Just try to setup AM.
It never gets any value/data

**Debug Information**:

**Comment by MatteoBiscosi** [comment]
hi @alexolivan which kind of active monitoring type did you configure?

**Comment by alexolivan** [comment]
Hi.

Initially, as I was struggling to have any AM results, I just simply setup a ICMP ping to 8.8.8.8

However, although certainly not sure, I feel that the kind of monitoring type shouldn't matter in this case, since the issue is that the round() function, used by the code on the script, is not 'included' from the the appropriate 'utils' lib.

Initially I just searched for the round() function, and I found that it sat at some 'utils' script (don't recall which one). Then took a look at other similar scripts calling round(), and I confirmed they were 'requiring' some of those util/libs scripts.
So I simply 'copy/pasted' the 'requiring' stuff somewhere at am_utils.lua , and voila! ...that was it.
If round() at am_utils.lua is to be found somehow other way, then, that way is not working,
 
Cheers.
